docs/reference/modules/scripting.asciidoc

```diff
@@ -78,8 +78,8 @@ supported scripting languages:
 |expression |yes       |built-in
 |mustache   |yes       |built-in
 |mvel       |no        |https://github.com/elastic/elasticsearch-lang-mvel[elasticsearch-lang-mvel]
-|javascript |no        |https://github.com/elastic/elasticsearch-lang-javascript[elasticsearch-lang-javascript]
-|python     |no        |https://github.com/elastic/elasticsearch-lang-python[elasticsearch-lang-python]
+|javascript |no        |https://github.com/elastic/elasticsearch/tree/master/plugins/lang-javascript[elasticsearch-lang-javascript]
```

Comment:
I'd link to the plugin documentation instead, eg: for https://www.elastic.co/guide/en/elasticsearch/plugins/current/lang-javascript.html

    | javascript | no | {plugins}/lang-javascript.html[elasticsearch-lang-javascript]